lightly/api/serve.py

```diff
@@ -0,0 +1,77 @@
+import re
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+from pathlib import Path
+from typing import Sequence
+
+
+def get_server(
+    paths: Sequence[str],
+    host: str,
+    port: int,
+):
+    """Returns an HTTP server that serves a local datasource.
+
+    Args:
+        paths:
+            List of paths to serve.
+        host:
+            Host to serve the datasource on.
+        port:
+            Port to serve the datasource on.
+
+    Examples:
+        >>> from lightly.api import serve
+        >>> from pathlib import Path
+        >>> serve(
+        >>>    paths=[Path("/input_dir), Path("/lightly_dir)],
+        >>>    host="localhost",
+        >>>    port=1234,
```

Comment:
best to use 4567 as example as afaik thats also the default in `set_local_datasource`

Reply:
3456